main.nf

```diff
@@ -338,70 +338,35 @@ process reverseStrand {
 
     input:
         path "reads_aligned.sorted.bam" from reads_sorted_bam
+        path "genome.fa" from params.newgenome
 
     output:
-        path "old_ref_alleles.txt" into old_ref_alleles
         path "variants_remapped.vcf" into variants_remapped
 
     """
     # Ensure that we will use the reverse_strand.py from this repo
-    ${baseDir}/variant_remapping_tools/reverse_strand.py -i reads_aligned.sorted.bam \
-        -p old_ref_alleles.txt -o variants_remapped.vcf -f $params.flankingseq \
-        -s $params.scorecutoff -d $params.diffcutoff
+    ${baseDir}/variant_remapping_tools/reads_to_remapped_variants.py -i reads_aligned.sorted.bam \
```

Comment:
In line with this rename, I suggest also renaming the Nextflow step (`reverseStrand`) and updating its description as well.